Week1/homework/app.js

```diff
@@ -1,11 +1,153 @@
+// Replace with your own book titles
+
 'use strict';
 
 {
   const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
+    'parallel-worlds',
+    'masnavi',
+    'black_holes_baby_universes',
+    'blindness',
+    'one_Hundred_years_solitude',
+    'girl_train',
+    'habits_highly_effective_people',
+    'philosophy_fear',
+    'sapiens',
+    'lust_life',
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
+  function makeList() {
+    const body = window.document.getElementById('body');
+
+    const ul = window.document.createElement('ul');
+
+    for (let i = 0; i < bookTitles.length; i++) {
+      const li = window.document.createElement('li');
+
+      ul.appendChild(li);
+      li.innerHTML = li.innerHTML + bookTitles[i];
+    }
+    body.appendChild(ul);
+  }
+  window.onload = () => makeList();
+}
+{
```

Comment:
Just wanted to confirm - is the reason of having two different block scopes because you wanted to have functions with the same name `makeList` or did you have a different reason for it?